packages/core/strapi/lib/services/entity-service/types/params/pagination.d.ts

```diff
@@ -0,0 +1,11 @@
+export type PageNotation = {
+  page?: number;
+  pageSize?: number;
+};
```

Comment:
Just curious, the [ES pagination documentation](https://docs.strapi.io/dev-docs/api/entity-service/order-pagination#pagination) only shows OffsetNotation.

Is PageNotation deprecated?

Reply:
It's supported in the API, should be documented :slightly_smiling_face: